fastplotlib/graphics/image.py

```diff
@@ -272,7 +274,8 @@ def __init__(
         start_ixs = [list(map(lambda c: c * chunk_size, chunk)) for chunk in chunks]
         stop_ixs = [list(map(lambda c: c + chunk_size, chunk)) for chunk in start_ixs]
 
-        self._world_object = pygfx.Group()
+        world_object = pygfx.Group()
```

Comment:
why do we want to expose the `world_object` now when it was always private before?